app/components/pages/Onboarding.tsx

```diff
@@ -80,6 +83,9 @@ export default class OnboardingPage extends TsxComponent<{}> {
       ) {
         this.stepsState.push({ complete: false });
       }
+      if (this.onboardingService.isTwitchAuthed) {
```

Comment:
This is a little confusing, but if this is the logic that enables the multistream step can we change this to check `restreamService.canEnableRestream` instead?  I know the only thing it checks right now is whether the user is Twitch authed, but I think it's good to have all restream UI looking at the same getter.

Reply:
i guess that makes sense, basically futureproofing against expanding restream to other platforms?